jiant/models.py

```diff
@@ -303,11 +304,11 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
         d_word = args.d_word
         word_embs = nn.Embedding(n_token_vocab, d_word).weight
     else:
-        assert args.input_module.startswith("bert") or args.input_module in [
-            "gpt",
-            "elmo",
-            "elmo-chars-only",
-        ], "You do not have a valid value for input_module."
+        assert (
+            args.input_module.startswith("bert")
+            or args.input_module.startswith("xlnet")
+            or args.input_module in ["gpt", "elmo", "elmo-chars-only"]
```

Comment:
not to pile more onto this PR, but do we get GPT-2 support for free?

Reply:
Not quite free, since each `pytorch-transformers` model class has its own idiosyncrasies but it should be pretty easy. Near future PR?